### PR TITLE
Format numerical text display

### PR DESCRIPTION
**Pull request by mmcfarland**
* Allow the app to function without Google availability
* Output values have comma's and decimal points, utilize the existing template formatting options.

Connects #678

### CODE REVIEW

**Comment by caseypt** [comment]
Maybe another issue, but consider verifying with Stroud or maybe @jamesmcclain how many significant digits the model results are valid to so that we can avoid any false precision issues.

Specifically thinking of the water quality output:

screen shot 2015-08-24 at 4 21 58 pm

**Comment by caseypt** [comment]
+1 (once the tests are good). All the numbers look properly formatted.

File: src/mmw/js/src/analyze/views.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ var TableRowView = Marionette.ItemView.extend({
         return {
             // Convert coverage to percentage for display.
             coveragePct: (this.model.get('coverage') * 100).toFixed(1),
-            areaTrimmed: this.model.get('area').toFixed(1)
+            area: this.model.get('area')
```
Comment (caseypt) [inline src/mmw/js/src/analyze/views.js]: I don't think we need this anymore since the model should be passed to the view automatically.

**Comment by jamesmcclain** [comment]
I am not sure how many of the digits are usable.

I would guess that it is no more than the number of digits found in the input (e.g. 3.70 inches of rain implies no more than two digits of precision after the decimal point in the results).  That is just a guess.

File: src/mmw/js/src/modeling/tr55/quality/templates/tableRow.html [hunk 2]
```diff
@@ -1,2 +1,2 @@
 <td>{{ measure }}</td>
-<td class="strong text-left">{{ roundedLoad }}</td>
+<td class="strong text-left">{{ load|round(3)|toLocaleString }}</td>
```
Comment (lliss) [inline src/mmw/js/src/modeling/tr55/quality/templates/tableRow.html]: Much nicer.

**Comment by lliss** [comment]
+1. Looks like the failing tests are just looking for known output of the numeric strings:

```
not ok 3 Firefox 39.0 - Analyze DetailsView renders tables that match the data when there are 1 categories in the dataset
    ---
        message: >
            expected [ 'Developed: Open', '5041.0', '26.3' ] to deeply equal [ 'Developed: Open', '5,041', '26.3' ]
```

and 

```
not ok 5 Firefox 39.0 - Analyze DetailsView renders tables that match the data when there are 2 categories in the dataset
    ---
        message: >
            expected [ 'Water', '21.0', '1.0' ] to deeply equal [ 'Water', '21', '1.0' ]
```

**Comment by lliss** [comment]
I'd favor rounding to two places.

**Comment by mmcfarland** [comment]
@azavea-bot rebuild

**Pull request merged**